From 12229f2a75bd1928d97797196b3364aee8158478 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Fri, 1 Dec 2017 11:57:51 +0000
Subject: [PATCH 7/8] lineage-sdk: Add broadcast action for power menu update

This action used to be defined in the frameworks, but can be placed
here from now on.

Change-Id: I7886c682aea97be40d2dcd0b8f147acb660193ab
---
 sdk/src/java/lineageos/content/Intent.java | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/sdk/src/java/lineageos/content/Intent.java b/sdk/src/java/lineageos/content/Intent.java
index 6f4f925..3a0ff95 100644
--- a/sdk/src/java/lineageos/content/Intent.java
+++ b/sdk/src/java/lineageos/content/Intent.java
@@ -109,4 +109,15 @@ public class Intent {
      */
     public static final String EXTRA_LID_STATE =
             "lineageos.intent.extra.LID_STATE";
+
+    /**
+     * Broadcast Action: Update preferences for the power menu dialog. This is to provide a
+     * way for the preferences that need to be enabled/disabled to update because they were
+     * toggled elsewhere in the settings (ie screenshot, user switcher, etc) so we don't have
+     * to do constant lookups while we wait for the menu to be created. Getting the values once
+     * when necessary is enough.
+     *@hide
+     */
+    public static final String ACTION_UPDATE_POWER_MENU =
+            "lineageos.intent.action.UPDATE_POWER_MENU";
 }
-- 
2.7.4

